plugins/sql_config_validator.h

```diff
@@ -168,6 +180,14 @@ template<typename FieldType> struct PlainColumnSelect
                     auto booleanExpression = getBooleanExpression(computedVariable->expression);
                     if (booleanExpression)
                         continue;
+                    // function(args)[extract]
+                    auto functionCallExpression = getFunctionCallExpression(computedVariable->expression);
+                    if (functionCallExpression)
+                        continue;
+                     // 1.0
+                    auto constantExpression = getConstantExpression(computedVariable->expression);
+                    if (constantExpression)
+                        continue;
```

Comment:
I came to the same conclusion but I did not want to include too many refactorings in this PR.